tools/pypy-test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+apt-get -yq update
+apt-get -yq install libatlas-dev libatlas-base-dev liblapack-dev
```

Comment:
Is PyPy allergic to OpenBLAS? We can deal with the linear algebra backend details later I suppose; I think the other Azure entries should now all be using wheel-matched OpenBLAS

Reply:
Adding installation of `https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas-v0.3.5-manylinux1_x86_64.tar.gz`. Should the travis tests be using this too?

Reply:
I don't know--I guess having some jobs that don't use OpenBLAS has value. It is hard to win here--if I try to improve the OpenBLAS usage / infrastructure people can just argue that there are other backends, or that we can swap them, and so on.. Anyway, looks like you did the right thing here, thanks!